manifests/database/mysql.pp

```diff
@@ -28,9 +28,9 @@
   assert_private()
 
   #
-  # Adjustments for version 3.0/4.0 - structure of package with sqls differs from previous versions
+  # Adjustments for version 3.x/4.x/5.x - structure of package with sqls differs from previous versions
   case $zabbix_version {
-    /^(3|4).\d+$/: {
+    /^[345].\d+$/: {
```

Comment:
if you like you could also just remove the case statement. we really don't support zabbix 1/2 anymore :D

Reply:
True :D

Reply:
mhm since some tests still run on 2.x, this might be something for a cleanup PR.

Reply:
I mean if i remove all the legacy code the PR could get pretty big